blockbase/sass/post/_meta.scss

```diff
@@ -1,4 +1,6 @@
 .post-meta {
+	row-gap: var(--wp--custom--gap--baseline) !important;
+
```

Comment:
Might make sense to add a code comment here to explain why we're using `!important`. And we can also probably remove that empty line below.